### PR TITLE
Reimplement Bootstrap Affix

### PR DESCRIPTION
**Pull request by stevenjoezhang**

### CODE REVIEW

**Comment by stevenjoezhang** [comment]
Ignore `Do not use 'new' for side effects` warning.

File: source/js/schemes/pisces.js [hunk 1]
```diff
@@ -1,132 +1,85 @@
...
-  const initAffix = () => {
-    let headerOffset = getHeaderOffset();
-    let footerOffset = getFooterOffset();
-    $('.sidebar-inner').affix({
-      offset: {
-        top   : headerOffset - sidebarOffset,
-        bottom: footerOffset
-      }
-    });
-    document.querySelector('#sidebar').css({
```
Comment (ivan-nginx) [inline source/js/schemes/pisces.js]: Can ID may be faster to find than class?

Comment (stevenjoezhang) [reply source/js/schemes/pisces.js]: Good question, I have not investigated it yet, but I believe this is much more efficient than jquery.

Comment (ivan-nginx) [reply source/js/schemes/pisces.js]: I'm not about Jquery, this must work in CSS.

**Comment by stevenjoezhang** [comment]
See also https://github.com/next-theme/hexo-theme-next/commit/b9968b6be0a8b20be80adad185ee94d713672e52

**Pull request merged**